ops/docker-compose.yml

```diff
@@ -179,6 +189,10 @@ services:
       ROLLUP_VERIFIER_ENABLE: 'true'
       ETH1_CTC_DEPLOYMENT_HEIGHT: 8
       RETRIES: 60
+
+      # must match l2geth environment
+      SYSTEM_ADDRESS_0_DEPLOYER: "0xa961b0d6dce82db098cf70a42a14add3ee3db2d5"
+      SYSTEM_ADDRESS_1_DEPLOYER: "0xdfc82d475833a50de90c642770f34a9db7deb725"
```

Comment:
They keys and addresses each appear multiple times in this file (if comments are counted). I think we should consider this approach to defining variables: 
- [compose docs](https://github.com/compose-spec/compose-spec/blob/master/spec.md#extension)
- relevant [SE question](https://stackoverflow.com/a/48651071/7230551)